### PR TITLE
SNO+: Minor fixes and added extra parameters to ECAs

### PR DESCRIPTION
**Pull request by jcaravaca**
Author: Javier Caravaca <jcaravaca@berkeley.edu>

### CODE REVIEW

**Comment by jcaravaca** [comment]
Tested at the detector.

File: Source/Experiments/SNOP/ECARun.m [hunk 1]
```diff
@@ -240,8 +263,12 @@ - (void) doECAs
         NSLog(@"Finishing ECA Run \n");
         NSLog(@"************************* \n");
 
-        //Unset pedestal masks
+        //Unset settings
         [aSNOPModel zeroPedestalMasks];
+        [anMTCModel setCoarseDelay:ECA_COARSE_DELAY]; //UNCOMMENT THIS LINE BEFORE COMMISSIONING
+        [anMTCModel loadCoarseDelayToHardware]; //UNCOMMENT THIS LINE BEFORE COMMISSIONING
+        [anMTCModel setFineDelay:ECA_FINE_DELAY]; //UNCOMMENT THIS LINE BEFORE COMMISSIONING
+        [anMTCModel loadFineDelayToHardware]; //UNCOMMENT THIS LINE BEFORE COMMISSIONING
```
Comment (tlatorre-uchicago) [inline Source/Experiments/SNOP/ECARun.m]: Need to reload the original settings here and not these hardcoded constants.

File: Source/Experiments/SNOP/ECARun.m [hunk 2]
```diff
@@ -190,18 +195,24 @@ - (void) doECAs
...
         if(![self triggersOFF]){
             goto stop;
         }
 
+        /* Get previous settings */
+        prev_coarsedelay = [anMTCModel coarseDelay];
+        prev_finedelay = [anMTCModel fineDelay];
+
+        /* Set MTC correct settings for ECAs */
+        [anMTCModel stopMTCPedestalsFixedRate];
```
Comment (tlatorre-uchicago) [inline Source/Experiments/SNOP/ECARun.m]: these need to be done on the main thread

File: Source/Experiments/SNOP/ECARun.m [hunk 3]
```diff
@@ -190,18 +195,24 @@ - (void) doECAs
...
-        }
-
         /* Disable channel triggers and stop if it fails */
         if(![self triggersOFF]){
             goto stop;
         }
 
+        /* Get previous settings */
+        prev_coarsedelay = [anMTCModel coarseDelay];
+        prev_finedelay = [anMTCModel fineDelay];
```
Comment (tlatorre-uchicago) [inline Source/Experiments/SNOP/ECARun.m]: also need to reset pedestal width.

**Comment by tlatorre-uchicago** [comment]
merged.

**Pull request closed without merging**